use-timescale/ha-replicas/read-scaling.md

```diff
@@ -1,79 +1,115 @@
 This is particularly useful when read traffic is very spiky and risks impacting ingest performance, or where reads have 
 a lower priority to writes. 
 
-This page shows you how to create and manage read replicas.
+This page shows you how to create and manage $READ_REPLICA sets in $CONSOLE.
 
 ## What is read replication?
 
-A read replica is a read-only copy of the primary data instance in your Timescale Cloud service. Queries on read 
-replicas have minimal impact on the performance of the primary data instance. This enables you to interact with 
-up-to-date production data for analysis or to scale out reads beyond the limits of your primary data instance. You use 
-read replicas for read scaling. To limit data loss for your Timescale Cloud services, use [High availability][ha].
+A $READ_REPLICA is a read-only copy of your primary data instance. Queries on $READ_REPLICA have minimal impact on the performance of the primary instance. This enables you to interact with up-to-date production data for analysis, or to scale out reads beyond the limits of your primary instance. $READ_REPLICA_CAPs can be short-lived and deleted when a session of data analysis is complete, or long-running to power a business intelligence tool. 
 
-You can create as many read replicas as you need. Each read replica appears as its own service. You use a unique
-connection string to interact with each read replica. This provides both security and resource isolation. To restrict 
-access without isolation, you can create a [read-only role][read-only-role] for each Timescale Cloud service. Users 
-with read-only permissions cannot access the primary data instance directly.
+A $READ_REPLICA set in $CLOUD_LONG is a group of one or more $READ_REPLICA nodes that share the same endpoint. You query each set as a single replica. $CLOUD_LONG balances the load between the nodes in the set for you.
 
-## Create a read replica
+You can create as many $READ_REPLICA sets as you need. For security and resource isolation, each $READ_REPLICA set has unique connection details.
 
-Read replicas can be short-lived and deleted when the analysis is complete, or long-running to power a
-business intelligence (BI) tool. To create a secure read replica for your read-intensive apps: 
+You use $READ_REPLICA sets for horizontal **read** scaling. To limit data loss for your $SERVICE_LONGs, use [$HA_REPLICAs][ha].
+
+## Prerequisites
+
+To follow this procedure:
+
+- Create a target $SERVICE_LONG.
+- Create a [read-only user][read-only-role] on the primary data instance. 
+
+  A user with the read-only permissions cannot access the primary data instance directly. This user is propagated to the $READ_REPLICA set when you create them.
```

Comment:
> A user with the read-only permissions cannot access the primary data instance directly.
This is not correct. All users are created in the primary and propagated to the replicas, including read-only users. A read-only user will not be able to make data changes on to the primary obviously.